src/commands/console/workspace/download.js

```diff
@@ -18,28 +19,38 @@ class DownloadCommand extends ConsoleCommand {
   async run () {
     await this.initSdk()
     try {
+      const errorMessage = []
       aioConsoleLogger.debug('Trying to fetch workspace configs')
+      const { args } = this.parse(DownloadCommand)
 
       const org = this.getConfig(ConsoleCommand.CONFIG_KEYS.ORG)
       if (!org) {
-        throw new Error('No Organization selected')
+        errorMessage.push('Organization')
       }
 
       const project = this.getConfig(ConsoleCommand.CONFIG_KEYS.PROJECT)
       if (!project) {
-        throw new Error('No Project selected')
+        errorMessage.push('Project')
       }
 
       const workspace = this.getConfig(ConsoleCommand.CONFIG_KEYS.WORKSPACE)
       if (!workspace) {
-        throw new Error('No Workspace selected')
+        errorMessage.push('Workspace')
+      }
+
+      if (errorMessage.length > 0) {
```

Comment:
it would be good that the output is the same for missing selections in org, project and workspace I don't have a strong opinion on how the error handling needs to be done but we should make it standard IMO
cc @rajarju

Reply:
We could print these errors in separate lines
Or if we want to show better messaging, we can check for multiple conditions

Reply:
@moritzraho It cannot be totally same as or/prj commans, both of them depend only on one obj but workspace needs three. So message cannot be exactly same.

Reply:
👍 thanks @rajarju  and @sandeep-paliwal, I am taking care of alignment will raise a PR soon